### ISSUE TITLE
[Feature]: support window controls overlay

### ISSUE THREAD

**Issue by Legend-Master** [MicrosoftEdge/WebView2Feedback#4532]
### Describe the feature/enhancement you need

Able to display content in the title bar

> https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/window-controls-overlay

### The scenario/use case where you would use this feature

Same as described here https://learn.microsoft.com/en-us/microsoft-edge/progressive-web-apps-chromium/how-to/window-controls-overlay

This feature is supported by Chromium (and Edge) for a while now, but it's still missing in WebView2

### How important is this request to you?

Impactful. My app's user experience would be significantly compromised without it.

**Comment by clearlysid** [comment]
Same request^. Having the right things exposed for WebView2 will significantly improve the UI for my app too!

**Comment by ChrisPulman** [comment]
If your request is related to a WPF application, then please take a look at this https://github.com/MicrosoftEdge/WebView2Feedback/issues/286#issuecomment-1712379863 it may help you to solve the issue in the short term, I know that it's not a solution to every situation but many people have been successful.

**Comment by Legend-Master** [comment]
No, it's for building applications with pretty much WebView2 only, like Electron but with the WebView provided by the OS

**Comment by victorhuangwq** [comment]
Hi @tochukwuIbeEkeocha is this similar to the work that you already doing on WCO?

**Comment by tochukwuIbeEkeocha** [comment]
Hello, yes I am currently working on adding this feature to Webview2.

**Comment by victorhuangwq** [comment]
Okay @Legend-Master we will track this issue in our backlog. At the same time, as @tochukwuIbeEkeocha has pointed out, he has already been working on it. 

We will provide any further updates using this thread.

**Comment by Legend-Master** [comment]
That's great! 😃

**Comment by metkm** [comment]
No news about this?

**Comment by victorhuangwq** [comment]
There is an ongoing specification for the feature that you can find here: 
https://github.com/MicrosoftEdge/WebView2Feedback/pull/4613/files?short_path=72a4d90#diff-72a4d90b49c01ab981142a20635948a83ba4366a75f4f59629b33f949be6b06d

**Comment by brucewedding** [comment]
Any update on this? I saw it went out in 1.0.3415-prerelease but it has yet to be promoted.

**Comment by tuyangJs** [comment]
没有什么消息了吗
